omnibus/config/projects/chef-workstation.rb

```diff
@@ -67,17 +67,18 @@
 dependency "rust-uninstall"
 
 dependency "chef-dk"
+dependency "chef-dk-gem-versions"
 
 dependency "gem-permissions"
 dependency "rubygems-customization"
+dependency "shebang-cleanup"
```

Comment:
I'm so sad it took me a week to figure this out. Any idea why we did not include this dependency originally @jonsmorrow @marcparadise @robbkidd ?